src/Faker/Provider/ru_RU/Person.php

```diff
@@ -154,4 +154,23 @@ public function middleName($gender = null)
             static::GENDER_FEMALE,
         )));
     }
+
+    /**
+     * Return last name for the specified gender.
+     *
+     * @access public
+     * @param string|null $gender A gender of the last name should be generated
+     *     for. If the argument is skipped a random gender will be used.
+     * @return string Last name
+     */
+    public function lastName($gender = null)
+    {
+        $lastName = static::randomElement(static::$lastName);
+        
+        if (static::GENDER_FEMALE === $gender) {
```

Comment:
Maybe we should add a link here for explanation?

For example, https://en.m.wikipedia.org/wiki/Eastern_Slavic_naming_customs#Grammar. 

/cc @pimjansen

Reply:
Agree, im not a fan of "only in our usecases" but it will do for no i guess